backend/src/main/scala/com/softwaremill/bootzooka/api/Routes.scala

```diff
@@ -28,20 +46,12 @@ trait Routes extends UsersRoutes
     } & handleRejections(rejectionHandler)
   }
 
-  val routes =
+  def base(route: Route) =
```

Comment:
Why do we need the split between "base routes" and "routes", and why do we need the extra RoutesBase trait?